app/views/api/guidance/test-environments.md

```diff
@@ -0,0 +1,20 @@
+## What are the test environments
+
+The test environments are used to test API integrations without affecting real data.
```

Comment:
do we want to add more detail here around bearer tokens or ECF testing envs? might be a repeat of existing content

Reply:
leave as is, might revisit in future